x/mint/types/params.go

```diff
@@ -168,3 +194,59 @@ func validateBlocksPerYear(i interface{}) error {
 
 	return nil
 }
+
+func validateGenesisEpochProvisions(i interface{}) error {
```

Comment:
Why do we not pass the data type directly here when func is not used anywhere else? I think that could remove type cast. Same as below funcs.